app/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdk 30
+    compileSdk 31
 
     defaultConfig {
         minSdkVersion 21
         //noinspection ExpiredTargetSdkVersion
-        targetSdkVersion 30
+        targetSdkVersion 31
```

Comment:
Do you think if we can keep this target SDK level 30? Compile SDK can be greater than 31, of course. The main thing is the animation stuff in https://github.com/appium/io.appium.settings/pull/174#issuecomment-2130875350

Reply:
Please ignore my ^ comment.
I think with https://github.com/appium/appium-espresso-driver/pull/1007 , the animation stuff affect only lower than api level 26, so the `@SuppressLint("SoonBlockedPrivateApi"` should be sufficient as this change